app/featuresmodel.cpp

```diff
@@ -150,3 +153,8 @@ Qt::ItemFlags FeaturesModel::flags( const QModelIndex &index ) const
 
   return Qt::ItemIsEditable;
 }
+
+int FeaturesModel::featuresCount() const
+{
+  return mFeatures.size();
```

Comment:
I am not sure if we do not want to display total number of features next to layer name? (@wonder-sk), not sure if displaying pagination limit is what user expects?

Reply:
@tomasMizera confirmed with Martin, should be total count of features in the layer

Reply:
done